Lib/xml/etree/ElementPath.py

```diff
@@ -157,6 +157,8 @@ def prepare_predicate(next, token):
             return
         if token[0] == "]":
             break
+        if token == ('', ''):
```

Comment:
Is it for whitespaces? Please add a comment.

Reply:
Done.